src/ConstructiveSolidGeometry/SurfacePrimitives/Plane.jl

```diff
@@ -114,6 +113,22 @@ function sample(tri::Plane{T, CartesianPoint{T}, Nothing}, step::Quantity{<:Real
     samples
```

Comment:
For instance here, the return value is a  `Vector{CartesianPoint{T}}`. We might run into conversion problems and rounding issues.